### PR TITLE
Allow to run refresh_scores by project and/or language

### LINKED ISSUE #6548: Allow to run refresh_scores by project
Allow to run `refresh_scores` command by project so it is easier/faster to debug.

### PR DESCRIPTION
**Pull request by unho**
Fixes #6548.

### CODE REVIEW

File: pootle/apps/pootle_score/updater.py [hunk 1]
```diff
@@ -347,8 +347,9 @@ class UserScoreUpdater(ScoreUpdater):
     store_score_model = UserStoreScore
     score_model = get_user_model()
 
-    def __init__(self, users=None, **kwargs):
+    def __init__(self, users=None, translation_project=None, **kwargs):
```
Comment (phlax) [inline pootle/apps/pootle_score/updater.py]: i would use `tp=` it saves a lot of typing and makes easier to keep < 80 chars

Comment (unho) [inline pootle/apps/pootle_score/updater.py]: I had that before :). Will revert.

File: pootle/apps/pootle_score/updater.py [hunk 2]
```diff
@@ -179,14 +179,14 @@ def create_scores(self, scores):
             objects=created)
         return created
 
-    def update(self, users=None, existing=None, date=None):
+    def update(self, users=None, existing=None, date=None, tp=None):
```
Comment (phlax) [inline pootle/apps/pootle_score/updater.py]: do we need this tp argument now?

File: pootle/apps/pootle_score/updater.py [hunk 3]
```diff
@@ -179,14 +179,14 @@ def create_scores(self, scores):
...
         if date is not None:
             start = date
             end = date + timedelta(days=1)
         else:
             start = end = None
         return self.set_scores(
-            self.calculate(users=users, start=start, end=end),
+            self.calculate(users=users, start=start, end=end, tp=tp),
```
Comment (phlax) [inline pootle/apps/pootle_score/updater.py]: ditto this one

File: pootle/apps/pootle_score/updater.py [hunk 4]
```diff
@@ -256,7 +256,7 @@ def score_event(self, event, calculated_scores):
                 calculated_scores[event_date][event.user.id].get(k, 0)
                 + score)
 
-    def calculate(self, start=None, end=None, users=None):
+    def calculate(self, start=None, end=None, users=None, **kwargs):
```
Comment (phlax) [inline pootle/apps/pootle_score/updater.py]: do we need these kwargs ?

**Comment by phlax** [comment]
lgtm, nice work @unho

**Pull request merged**